### PR TITLE
EJDOTNETCORE-4254 Added README file

### PR DESCRIPTION
**Pull request by jeyalakshmit**
Added README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
 # Handling images in PDF file using C#
```
Comment (ghost) [inline README.md:1]: Add intro about Syncfusion .NET PDF library

File: README.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
 PDF has become one of the most popular and widely used file formats, here images play an important role in PDF documents, images can grab our attention easily. So, in this article, you will learn how to add, extract, remove, and replace images from PDF documents using C#.
```
Comment (ghost) [inline README.md]: PDF has become one of the most popular and widely used file formats, here images play an important role in PDF documents. This repository contains examples to add, extract, remove and replace images from PDF documents using C#.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
 
-## Blog reference
-[Add, Remove, Extract, and Replace Images in PDF using C#](https://www.syncfusion.com/blogs/post/add-remove-extract-and-replace-images-in-pdf-using-csharp.aspx)
+Sample name | Description
+--- | ---
+[Add image](https://github.com/SyncfusionExamples/pdf_image_handling/blob/master/Add_image) | To add an image to a PDF file in console application.
```
Comment (ghost) [inline README.md]: Add an image in a PDF document

File: README.md [hunk 4]
```diff
@@ -1,5 +1,37 @@
...
+[Extract image](https://github.com/SyncfusionExamples/pdf_image_handling/blob/master/Extract_images) | To extract all the images from a PDF file in console application.
```
Comment (ghost) [inline README.md]: Extract images from an existing PDF document

File: README.md [hunk 5]
```diff
@@ -1,5 +1,37 @@
...
+[Remove image](https://github.com/SyncfusionExamples/pdf_image_handling/blob/master/Remove_images) | To remove images from a PDF document in console application.
```
Comment (ghost) [inline README.md]: Remove images from an existing PDF document

File: README.md [hunk 6]
```diff
@@ -1,5 +1,37 @@
...
 PDF has become one of the most popular and widely used file formats, here images play an important role in PDF documents, images can grab our attention easily. So, in this article, you will learn how to add, extract, remove, and replace images from PDF documents using C#.
 
-## Blog reference
-[Add, Remove, Extract, and Replace Images in PDF using C#](https://www.syncfusion.com/blogs/post/add-remove-extract-and-replace-images-in-pdf-using-csharp.aspx)
+Sample name | Description
+--- | ---
+[Add image](https://github.com/SyncfusionExamples/pdf_image_handling/blob/master/Add_image) | To add an image to a PDF file in console application.
+[Extract image](https://github.com/SyncfusionExamples/pdf_image_handling/blob/master/Extract_images) | To extract all the images from a PDF file in console application.
+[Remove image](https://github.com/SyncfusionExamples/pdf_image_handling/blob/master/Remove_images) | To remove images from a PDF document in console application. 
+[Replace image](https://github.com/SyncfusionExamples/pdf_image_handling/blob/master/Replace_image) | Replaced an existing image in a PDF document in console application.
```
Comment (ghost) [inline README.md]: Replace an existing image with a new image or modify an existing image in a PDF document.

File: README.md (hunk 1 above)
Comment (jeyalakshmit) [reply README.md:1]: Added introduction

File: README.md (hunk 2 above)
Comment (jeyalakshmit) [reply README.md]: changed the description

File: README.md (hunk 3 above)
Comment (jeyalakshmit) [reply README.md]: changed the description

File: README.md (hunk 4 above)
Comment (jeyalakshmit) [reply README.md]: changed the description

File: README.md (hunk 5 above)
Comment (jeyalakshmit) [reply README.md]: changed the description

File: README.md (hunk 6 above)
Comment (jeyalakshmit) [reply README.md]: changed the description

**Review by ghost — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -1,5 +1,37 @@
 # Handling images in PDF file using C#
-PDF has become one of the most popular and widely used file formats, here images play an important role in PDF documents, images can grab our attention easily. So, in this article, you will learn how to add, extract, remove, and replace images from PDF documents using C#.
+The [Syncfusion .NET PDF library](https://www.syncfusion.com/document-processing/pdf-framework/net) allows users to add and manipulate images in PDF files. PDF has become one of the most popular and widely used file formats, here images play an important role in PDF documents. This repository contains examples to add, extract, remove and replace images from PDF documents using C#.
```
Comment (ghost) [inline README.md]: The PDF has become one of the most popular and widely used file formats, here images play an important role in PDF documents.

Comment (jeyalakshmit) [reply README.md]: Changed the description

**Review by praveen-hari — APPROVED** [review]

**Pull request merged**